src/factories/CommandFactory.ts

```diff
@@ -20,7 +22,7 @@ class CommandFactory {
 			const aliases = new Command().getAliases();
 
 			aliases.forEach((alias: string) => {
-				this.commands[alias] = () => new Command();
+				this.commandAliases[alias] = () => new Command();
```

Comment:
I'm not entirely sure why this change was introduced, but it has broken the activation of commands through aliases. Is this still required? Seems as though `commandAliases` is not used anywhere

Reply:
I did this as I wanted to have two separate arrays: just commands and commands + aliases. This means that the embed (or any future use cases) does not need to do the filtering. I see what you mean about aliases no longer being triggered, though - will fix.